spec_cleaner/rpmpreambleelements.py

```diff
@@ -114,12 +115,17 @@ def __init__(self, options):
         self.license_conversions = options['license_conversions']
 
     def _sort_helper_key(self, a):
-        t = type(a)
-        if t == str:
+        if isinstance(a, str):
             key = a
-        elif t == list:
+        elif isinstance(a, RpmRequiresToken):
+            key = a.dump_token()
+        elif isinstance(a, list):
             # if this is a list then all items except last are comment or whitespace
-            key = a[-1]
+            output = a[-1]
+            if isinstance(output, str):
+                key = output
+            else:
+                key = output.dump_token()
```

Comment:
this is repeating over and over, what about providing `__str__()` ?

it would then look like
```
key = str(output)
```